docs/providers/aws/guide/serverless.yml.md

```diff
@@ -88,7 +88,7 @@ provider:
         userPoolArn: 'arn:aws:cognito-idp:us-east-1:123412341234:userpool/us-east-1_123412341', # required
         userPoolClientId: '1h57kf5cpq17m0eml12EXAMPLE', # required
         userPoolDomain: 'your-test-domain' # required
-        allowUnauthenticated: true # If set to true this allows the request to be forwarded to the target when user is not authenticated. Omit this parameter to make a HTTP 401 Unauthorized error be returned instead
+        onUnauthenticatedRequest: 'allow' # If set to 'allow' this allows the request to be forwarded to the target when user is not authenticated. When omitted it defaults 'deny' which makes a HTTP 401 Unauthorized error be returned. Alternatively configure to 'authenticate' to redirect request to IdP authorization endpoint.
```

Comment:
By convention for optional parameters we put _defaults_ as values. So let's use `'deny'` as value.